README.md

```diff
@@ -8,26 +8,26 @@ https://cdsvacpoc.azurewebsites.net/
 
 ## Quickstart
 
-* `yarn install`
```

Comment:
you don't like `*` apparently.

Reply:
I think `prettier` did that!

Reply:
huh - new prettier has different pretty-ness standards maybe?